package-lock.json

```diff
@@ -1633,14 +1633,6 @@
 				"acorn": "^6.0.0 || ^7.0.0 || ^8.0.0"
 			}
 		},
-		"node_modules/add-dom-event-listener": {
```

Comment:
I'm curious why there is a change in a package associated with this PR. This PR relates to a test case so this seems outside its scope. Should it be here?